documentation/writing_tasks.md

```diff
@@ -967,6 +967,67 @@ Successful on 1 target: localhost
 Ran on 1 target in 0.12 seconds
 ```
 
+### PowerShell example
+
+Create task and metadata in your project at
+`<PROJECT DIRECTORY>/tasks/mytask.{json,ps1}`.
+
+**Metadata**
+
+```json
+{
+  "files": [
+    "powershell_task_helper/files/BoltPwshHelper/"
+  ],
+  "input_method": "powershell"
+  "parameters": {
+    "name": {
+      "description": "Name or list of SCCM Trigger to run",
+      "type": "String"
+    }
+  }
+}
+```
+
+**Task**
+
+```powershell
+#!/usr/bin/env pwsh
+[CmdletBinding()]
+param (
+  # Name or list of SCCM Trigger to run
+  [Parameter(Mandatory=$true)]
+  [string]$Name
+)
+
+<#
+If using PowerShell 3.0, add an import statement here:
+Import-Module BoltPwshHelper
+#>
+​
+trap {
+  Write-BoltError -Message "Generic trap handler" -Exception $_
+}
```

Comment:
We should use the complete task and metadata example from https://github.com/jpogran/puppetlabs-powershell_task_helper/tree/pwsh-content/examples , since this example doesn't produce the output shown below.

Reply:
Updated